apps/server/src/controllers/integrationController.ts

```diff
@@ -222,6 +237,74 @@ export function dispatchFromAdapter(type: string, payload: unknown, source?: 'os
       return { topic: 'timer', payload: timer };
     }
 
+    // ontime/change/{eventID}/{propertyName}
+    case 'change': {
+      try {
+        if (params.length < 2) {
+          throw new Error('Invalid parameters');
+        }
+
+        const eventID = params[0];
+        const propertyName = params[1] as keyof OntimeEvent;
+
+        const isKeyOfOntimeEvent = (key: string): key is keyof OntimeEvent => {
+          return key in event;
+        };
+
+        if (!isKeyOfOntimeEvent(propertyName)) {
+          throw new Error(`Cannot update unknown event property ${propertyName}`);
+        }
+
+        const acceptedPropertyPayloadMap = {
+          title: 'string',
+          subtitle: 'string',
+          presenter: 'string',
+          note: 'string',
+          cue: 'string',
+
+          duration: 'number',
+
+          isPublic: 'boolean',
+          skip: 'boolean',
+
+          colour: 'string',
+          user0: 'string',
+          user1: 'string',
+          user2: 'string',
+          user3: 'string',
+          user4: 'string',
+          user5: 'string',
+          user6: 'string',
+          user7: 'string',
+          user8: 'string',
+          user9: 'string',
+        };
```

Comment:
we would likely prefer not having this being declared in the closure scope. maybe a config file? like `integrationController.config.ts`? Better ideas welcome